designer_v2/lib/features/design/study_form_controller.dart

```diff
@@ -145,6 +146,7 @@ class StudyFormViewModel extends FormViewModel<Study>
     enrollmentFormViewModel.buildFormData().apply(studyCopy);
     measurementsFormViewModel.buildFormData().apply(studyCopy);
     interventionsFormViewModel.buildFormData().apply(studyCopy);
+    reportsFormViewModel.buildFormData().apply(studyCopy);
```

Comment:
Not related to the bug I guess, but why is this not included here? @ibrahimozkn any idea?